test/fixtures/section-heading-first-fail.html

```diff
@@ -0,0 +1,3 @@
+<section>
+    <p>Hello, world.</p>
+</section>
```

Comment:
I mean, on the html page, you should have something like 
```
<section>
    <p>Hello, world.</p>
</section>
<section>
   <h1>foo</h1>
</section>
```
To make sure that when there is a passing instance (1), we still find the failing instance (1 failure).